docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -26,98 +26,164 @@ Sometimes we just need to reboot the jetson device when it gets stuck in initial
 ### Cuda
 
 The Cuda is installed by default while the cudnn is not if we use the first method. We have to write the cuda path and lib to `$PATH` and `$LD_LIBRARY_PATH`:
-```
+```shell
 export PATH=$PATH:/usr/local/cuda/bin
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64
 ```
 Then we can use `nvcc -V` the get the version of cuda we use.
 
-### Anaconda
+### Conda
 
 We have to install [Archiconda](https://github.com/Archiconda/build-tools/releases) instead as the Anaconda does not provide the wheel built for jetson.
+```shell
+wget https://github.com/Archiconda/build-tools/releases/download/0.2.3/Archiconda3-0.2.3-Linux-aarch64.sh
+bash Archiconda3-0.2.3-Linux-aarch64.sh
+source ~/.bashrc
+conda --version
+```
 
-After we installed the Archiconda successfully and created the virtual env correctly. If the pip in the env does not work properly or throw `Illegal instruction (core dumped)`, we may consider re-install the pip manually, reinstalling the whole JetPack SDK is the last method we can try.
+After we installed the Archiconda successfully, we can create the virtual env `mmdeploy` using the command below.
+```shell
+conda create -n mmdeploy python=3.6 # must be python 3.6
+conda activate mmdeploy
+```
```

Comment:
Well, the python version depends on versions of tensorrt python package pre-installed by Jetpack. So, maybe it is not a must here for Xavier or TX2 users.

Reply:
Done